app/queries/admin/moderators_query.rb

```diff
@@ -4,7 +4,7 @@ class ModeratorsQuery
       state: :trusted
     }.with_indifferent_access.freeze
 
-    VALID_ROLES = %i[banned warned trusted comment_banned].freeze
+    VALID_ROLES = %i[suspended warned trusted comment_suspended].freeze
```

Comment:
Since `user.suspended` still looks for the `banned` role in the DB during the transition, should this query still keep `banned` and `comment_banned` as valid roles and remove them in a following PR?

Reply:
@rhymes See https://github.com/forem/forem/pull/12270#discussion_r562338386